README.md

```diff
@@ -373,6 +373,42 @@ docker run -d -p 80:80 -p 443:443 \
 
 You'll need apache2-utils on the machine where you plan to create the htpasswd file. Follow these [instructions](http://httpd.apache.org/docs/2.2/programs/htpasswd.html)
 
+### Upstream (Backend) Server HTTP Load Balancing Support
+
+> **Warning**
+> This feature is experimental.  The behavior may change (or the feature may be removed) without warning in a future release, even if the release is not a new major version.  If you use this feature, please provide feedback so we know whether it is working properly and can be promoted to officially supported.
```

Comment:
I'll have the same request as on https://github.com/nginx-proxy/nginx-proxy/pull/1934 : please open a discussion for feedbacks on this feature first, then include a link to it in the documentation.

Reply:
Done. #2195 
And also squashed.